arch/arm/soc/st_stm32/stm32f0/Kconfig.defconfig.series

```diff
@@ -24,7 +24,10 @@ endif # GPIO_STM32
 
 if I2C && (I2C_1 || I2C_2 || I2C_3)
 
-config  I2C_STM32_V2
+config I2C_STM32
```

Comment:
We should move this in arch/arm/soc/st_stm32/common/Kconfig.defconfig.series
```
if I2C && (I2C_1 || I2C_2 || I2C_3)
config I2C_STM32
	def_bool y
endif # I2C
```
and leave here
```
if I2C_STM32
config I2C_STM32_V2
 	def_bool y
endif # I2C_STM32
```